--- Python-2.7.2/Lib/cgi.py-local	2012-01-16 21:05:45.851988493 -0800
+++ Python-2.7.2/Lib/cgi.py	2012-01-16 21:06:38.803316979 -0800
@@ -1,10 +1,6 @@
-#! /usr/local/bin/python
+#! /usr/bin/env python
 
-# NOTE: the above "/usr/local/bin/python" is NOT a mistake.  It is
-# intentionally NOT "/usr/bin/env python".  On many systems
-# (e.g. Solaris), /usr/local/bin is not in $PATH as passed to CGI
-# scripts, and /usr/local/bin is the default directory where Python is
-# installed, so /usr/bin/env would be unable to find python.  Granted,
+# ....  Granted,
 # binary installations by Linux vendors often install Python in
 # /usr/bin.  So let those vendors patch cgi.py to match their choice
 # of installation.
