--- Python-2.7.4/Modules/getbuildinfo.c	2013-04-06 07:02:37.000000000 -0700
+++ Python-2.7.4-notimestamp/Modules/getbuildinfo.c	2013-05-09 11:04:08.960671214 -0700
@@ -4,21 +4,9 @@
 #include <stdio.h>
 #endif
 
-#ifndef DATE
-#ifdef __DATE__
-#define DATE __DATE__
-#else
-#define DATE "xx/xx/xx"
-#endif
-#endif
+#define DATE "May 09 2013"
 
-#ifndef TIME
-#ifdef __TIME__
-#define TIME __TIME__
-#else
-#define TIME "xx:xx:xx"
-#endif
-#endif
+#define TIME "12:34:56"
 
 /* on unix, SVNVERSION is passed on the command line.
  * on Windows, the string is interpolated using
